[COMSUP-139] Add note for Astro 2.x support only (#896)
edgio-docs/edgio-docs: src/guides/v7/sites_frameworks/getting_started/astro.md

[Astro](https://astro.build/) is a modern static site builder. This guide walks you through deploying Astro sites to {{ PRODUCT }}.

<Callout type="warning">

>   This guide and the {{ PRODUCT }} platform currently supports Astro 2.x. Using Astro 3.x is not supported due to Node.js requirements and will cause runtime errors.

Reviewer: ```suggestion
  {{ PRODUCT }} only supports Astro 2.x. Astro 3.x is unsupported due to Node.js requirements and its usage will generate runtime errors.
```
